bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```
@@ -106,6 +106,7 @@ thing-type.shelly.shellyplussmoke.description = Shelly Plus Smoke - Smoke Detect
 thing-type.shelly.shellypluswdus.description = Shelly Wall Dimmer US Device
 thing-type.shelly.shellyplus10v.description = Shelly Plus Dimmer 10V
 thing-type.shelly.shellyplusrgbwpm-color.description = Shelly Plus RGBW PM - RGBW Dimmer with Power Meter
+thing-type.shelly.shellyplusstrip.description = Shelly Power Strip 4 
```

Should it be:
```suggestion
thing-type.shelly.shellyplusstrip.description = Shelly Plus Power Strip 4 
```

as in the README?